### PR TITLE
SSCS-6809 Changes to support new PIP AE office added by DWP

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -84,6 +84,11 @@ data "azurerm_key_vault_secret" "robotics_email_scottish_to" {
   vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
 }
 
+data "azurerm_key_vault_secret" "robotics_email_pip_ae_to" {
+  name      = "robotics-email-pip-ae-to"
+  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
```
Comment (iguannago) [inline infrastructure/main.tf]: have you put this value in the Vault??

Comment (srochani) [reply infrastructure/main.tf]: No not yet put, I might need your help on this

File: src/main/java/uk/gov/hmcts/reform/sscs/domain/email/RoboticsEmailTemplate.java [hunk 2]
```diff
@@ -12,21 +12,27 @@
...
 
-    public Email generateEmail(String subject, List<EmailAttachment> attachments, boolean isScottish) {
+    public Email generateEmail(String subject,
+                               List<EmailAttachment> attachments,
+                               boolean isScottish,
+                               boolean isPipAeTo) {
         return new Email(
                 from,
-                isScottish ? scottishTo : to,
+                isPipAeTo ? pipAeTo : isScottish ? scottishTo : to,
```
[the remark this answers is not in the document]
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/domain/email/RoboticsEmailTemplate.java]: Fixed logic, now scottish check first then pip ae

File: infrastructure/main.tf (hunk 1 above)
Comment (srochani) [reply infrastructure/main.tf]: The key Vault is created on AAT and PROD

**Pull request merged**